README.md

```diff
@@ -24,36 +22,26 @@
 1. `yarn install`
 2. copy over the config `cp .env.local.example .env.local`
 3. `yarn dev` to run the repo
-4. (optional) `yarn storybook` to
+4. (optional) `yarn storybook` to see component examples
 
-## Install Cosmos-SDK
+## Install Regen Ledger
```

Comment:
I'm not why sure this diff is showing here - it's actually already the current state of `main`. I assume something wonky happened with git history when I recently rebased